### PR TITLE
Moneo refresh

### PR DESCRIPTION
**Pull request by rafsalas19**
- Update Templates
- bug fixes

### CODE REVIEW

File: linux_service/start_moneo_services.sh [hunk 1]
```diff
@@ -24,14 +25,16 @@ if lspci | grep -iq NVIDIA ; then
     procs+=("nvidia_exporter")
 fi
 
-if [[ -n $PublisherMethod ]]; then
+if [[ -n $PublisherMethod ]] ; then
```
Comment (LiquidPT) [inline linux_service/start_moneo_services.sh:28]: why did this line need a space, but the others below didn't?

**Review by LiquidPT — APPROVED** [review]

File: linux_service/start_moneo_services.sh [hunk 2]
```diff
@@ -69,6 +72,10 @@ sudo systemctl start moneo@node_exporter.service
 sudo systemctl start moneo@net_exporter.service
 sudo systemctl start moneo@nvidia_exporter.service
 
+if [ "$PublisherMethod" == "workers" ]; then
+    proc_check false
```
Comment (RyoYang) [inline linux_service/start_moneo_services.sh:76]: Need a `exit 1` here?

**Review by RyoYang — APPROVED** [review]

File: linux_service/start_moneo_services.sh (hunk 2 above)
Comment (rafsalas19) [reply linux_service/start_moneo_services.sh:76]: let me take a look

File: linux_service/start_moneo_services.sh (hunk 1 above)
Comment (rafsalas19) [reply linux_service/start_moneo_services.sh:28]: Good catch, let me address (for clarity the functionality does not change, just a formatting issue)

File: linux_service/start_moneo_services.sh (hunk 2 above)
Comment (rafsalas19) [reply linux_service/start_moneo_services.sh:76]: The proc check function will handle the exit conditions in this case

Comment (RyoYang) [reply linux_service/start_moneo_services.sh:76]: Make sense for me, thanks!

**Pull request merged**